# Compare this snippet from Class-5/exception.py:
print(10/0)
text = "Hello"
number = int(text)